### PR TITLE
[android] Hide enable/disable tts button in pedestrian routing mode.

### PR DESCRIPTION
**Pull request by ygorshenin**
@yunikkk or @trashkalmar PTAL
Not sure that this is a good fix.

### CODE REVIEW

**Comment by ygorshenin** [comment]
https://jira.mail.ru/browse/MAPSME-2037

File: android/src/com/mapswithme/maps/widget/menu/NavMenu.java [hunk 1]
```diff
@@ -110,5 +111,15 @@ public void show(boolean show)
   {
     super.show(show);
     measureContent(null);
+    
+    if (Framework.nativeGetRouter() == Framework.ROUTER_TYPE_PEDESTRIAN)
```
Comment (yunikkk) [inline android/src/com/mapswithme/maps/widget/menu/NavMenu.java]: Лучше просто UiUtils.showIf(Framework.nativeGetRouter() == Framework.ROUTER_TYPE_PEDESTRIAN, mTts)

Comment (ygorshenin) [inline android/src/com/mapswithme/maps/widget/menu/NavMenu.java]: Done

**Comment by ygorshenin** [comment]
@yunikkk PTAL

**Pull request merged**